packages/db/fauna/resources/Index/pinsContentByStatus.js

```diff
@@ -0,0 +1,36 @@
+import fauna from 'faunadb'
+
+const {
+  CreateIndex,
+  Collection,
+  If,
+  Index,
+  Exists,
+  Not
+} = fauna
+
+/**
+ * Usage:
+ *
+ * Match(
+ *   Index('pins_content_by_status'),
+ *   StatusString // Pinned, Pinning, PinQueued, ...
+ * )
+ */
+const index = {
+  name: 'pins_content_by_status',
+  source: Collection('Pin'),
+  terms: [
+    { field: ['data', 'status'] }
+  ],
+  values: [
+    { field: ['data', 'content'] }
```

Comment:
We have a pin object for each node a piece of content is pinned on. So if pinned on 3 nodes in the cluster then 3 pins and I assume 3 results for the same content here - they need to be deduplicated before suming.

I'm not sure if this is the right/scalable way to aggregate this information.

Reply:
Exactly. I was thinking the total sum of bytes pinned would be 3x the actual content if we have 3 pin locations. How is this in nft.storage?